CHANGELOG.md

```diff
@@ -1,5 +1,197 @@
 # Node.js ChangeLog
 
+## 2015-12-04, Version 5.1.1 (Stable), @rvagg
+
+### Notable changes
+
+* **http**: Fix CVE-2015-8027, a bug whereby an HTTP socket may no longer have a parser associated with it but a pipelined request attempts trigger a pause or resume on the non-existent parser, a potential denial-of-service vulnerability. (Fedor Indutny)
```

Comment:
I speculate that `attempts trigger` -> `attempts to trigger`
(sorry If I'm wrong)